sdk/eventhub/eventhubs-checkpointstore-tables/review/eventhubs-checkpointstore-blob.api.md

```diff
@@ -0,0 +1,28 @@
+## API Report File for "@azure/eventhubs-checkpointstore-blob"
```

Comment:
I think this file was generated before you renamed your project and can be deleted.